database-plugins/src/main/java/io/cdap/plugin/db/common/DBBaseConfig.java

```diff
@@ -68,8 +68,8 @@
   @Description("The existing connection to use.")
   private DBConnectorConfig connection;
 
-  public Boolean getEnableAutoCommit() {
-    return enableAutoCommit;
+  public boolean getEnableAutoCommit() {
```

Comment:
don't see the constructor is added, can you add it like this https://github.com/cdapio/hydrator-plugins/blob/50bb1436e040b27ab36f50b44e4d78299ba4fc20/database-plugins/src/main/java/io/cdap/plugin/ConnectionConfig.java#L92-L95

Reply:
as discussed offline, since we already change the return value to `boolean` , no need to add a constructor here.